ochk/resource_subtenant_test.go

```diff
@@ -0,0 +1,115 @@
+package ochk
+
+import (
+	"context"
+	"fmt"
+	"github.com/chmurakrajowa/terraform-provider-ochk/ochk/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"testing"
+)
+
+type SubtenantTestData struct {
+	ResourceName          string
+	Description           string
+	Email                 string
+	MemoryReservedSizeMB  int64
+	Name                  string
+	NetworkIDs            []string
+	StorageReservedSizeGB int64
+	UserIDs               []string
+}
+
+func (c *SubtenantTestData) ToString() string {
+	return executeTemplateToString(`
+resource "ochk_subtenant" "{{ .ResourceName}}" {
+	name = "{{.Name}}"
+	email = "{{.Email}}"
+	description = "{{.Description}}"
+	memory_reserved_size_mb = "{{.MemoryReservedSizeMB}}"
+	storage_reserved_size_gb = "{{.StorageReservedSizeGB}}"
+	users = {{ StringsToTFList .UserIDs}}
```

Comment:
a tu nie chciałeś zrobić fora template'owego? a co jeśli users będzie puste?

Reply:
Nie może być puste. Ale jakby nic nie podać, to zostanie zwrócone `users = []`. 
Nie chciałem fora template'owego, bo jest nieprzejrzysty. Do tego zrobienie ifa w środku na ominięcie przecinka na ostatnim elemencie czyni to niepraktyczne :( To nie jinja :(